curriculum/challenges/english/25-front-end-development/workshop-flexbox-photo-gallery/6153a3ebb4f7f05b8401b716.md

```diff
@@ -19,8 +19,11 @@ All nine of your `img` elements should have a `alt` attribute with text describi
 
 ```js
 const images = [...document.querySelectorAll('img')];
-assert(images.every(image => image.getAttribute('alt')));
-assert(images.every(image => image.getAttribute('alt').length >= 5));
+images.forEach((image, index) => {
+  const alt = image.getAttribute('alt');
+  assert.exists(alt);
+  assert.isAtLeast(alt.length, 5);
```

Comment:
```suggestion
images.forEach((image) => {
  const alt = image.getAttribute('alt');
  assert.isAtLeast(alt?.length, 5);
```

with optional chaining you don't need to check if a thing exists first, and `index` is not used, so you can not put it there